tests/polkadotjs_test/test/test-polkadot.js

```diff
@@ -98,7 +98,6 @@ describe('Testing polkadot.js/api calls:', function () {
                 });
         })
     });
-    //TODO: remove skip when rpc.state.queryStorage is fixed (in PR#2505)
     describe.skip('api query', () => {
```

Comment:
so should we unskip that one?

Reply:
No, this isn't working yet, re-added TODO